### PR TITLE
Tss 483 new priorty form rebased onto 463

### PR DESCRIPTION
**Pull request by Santino-Trade**

### CODE REVIEW

File: api/barriers/serializers/base.py [hunk 1]
```diff
@@ -61,6 +61,7 @@ class BarrierSerializerBase(
     last_seen_on = serializers.SerializerMethodField()
     modified_by = UserField(required=False)
     priority = BarrierPriorityField(required=False)
```
Comment (NickFitz) [inline api/barriers/serializers/base.py:63]: Looks like I missed this bit - does this need to go too?

File: api/barriers/serializers/csv.py [hunk 2]
```diff
@@ -51,6 +51,7 @@ class BarrierCsvExportSerializer(AssessmentFieldsMixin, serializers.Serializer):
     product = serializers.CharField()
     source = serializers.SerializerMethodField()
     priority = serializers.SerializerMethodField()
```
Comment (NickFitz) [inline api/barriers/serializers/csv.py:53]: Also missed this one…

File: api/barriers/serializers/data_workspace.py [hunk 3]
```diff
@@ -201,6 +201,7 @@ class Meta(BarrierSerializerBase.Meta):
             "modified_on",
             "other_source",
             "priority",
```
Comment (NickFitz) [inline api/barriers/serializers/data_workspace.py:203]: And another! 😱

**Review by NickFitz — CHANGES_REQUESTED** [review]

File: api/barriers/serializers/base.py (hunk 1 above)
Comment (Santino-Trade) [reply api/barriers/serializers/base.py:63]: Just checked, we've got to keep it in the csv and DW, so think we need to keep this stuff

Comment (NickFitz) [reply api/barriers/serializers/base.py:63]: No, would be the correct answer 🤦‍♂️

**Review by NickFitz — APPROVED** [review]

**Pull request merged**